x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/notes_tab.cy.ts

```diff
@@ -26,7 +26,12 @@ import { addNotesToTimeline, goToNotesTab } from '../../../tasks/timeline';
 
 import { deleteTimelines } from '../../../tasks/api_calls/common';
 
-const text = 'system_indices_superuser';
+const systemIndicesSuperuserName = 'system_indices_superuser';
```

Comment:
nit, I would capitalize user similar to how you do it for CloudUser
```suggestion
const systemIndicesSuperuserName  => const systemIndicesSuperUserName;
```

Reply:
Thanks!! That was a typo